app/assets/stylesheets/header-footer-only.scss

```diff
@@ -1,3 +1,6 @@
+$govuk-compatibility-govuktemplate: true;
+@import "govuk_publishing_components/all_components";
```

Comment:
What's the performance impact of this?

Reply:
Missed that line. It's quite bad I would imagine. @kr8n3r can we only fish out the bits we need? I imagine that would be setting and utilities.

Reply:
this is not going into master

publishing components gem doesn't expose anything other than all ?

Reply:
Yeah, aware, but still I think we can do better. `govuk_publishing_components` exposes all `govuk-frontend`, so we can do, for example, `@import  "govuk/settings/all"` in any application that includes the gem.

Reply:
ok, wasn't sure. will change the imports